doc/source/tune/faq.rst

```diff
@@ -733,3 +733,59 @@ If `grid_search` is provided as an argument, the grid will be repeated ``num_sam
 Note that search spaces may not be interoperable across different search algorithms.
 For example, for many search algorithms, you will not be able to use a ``grid_search`` or ``sample_from`` parameters.
 Read about this in the :ref:`Search Space API <tune-search-space>` page.
+
+.. _tune-working-dir:
+
+How do I access relative filepaths in my Tune training function?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's say you launch a Tune experiment from ``~/code/my_script.py``. By default, Tune
+changes the working directory of each worker from ``~/code`` to its corresponding trial
+directory (e.g. ``~/ray_results/exp_name/trial_0000x``). This guarantees separate working
+directories for each worker process.
+
+There are two main reasons for this default:
+
+1. If trial writes artifacts, we want to avoid conflicts: if two trials on the same machine write to ``artifact.bin``, the second one would overwrite the artifact from the first.
+
+2. In distributed settings with more than one machine, we cannot guarantee that the working directory from the driver script exists on the other remote nodes. However, we can guarantee that the trial directory exists.
+
+If you need to access files from the original working directory
+(ex: you need to load a (relatively small) dataset or
+saved models from ``~/code/data/data.csv`` or ``~/code/models/model.h5``),
+we recommend you to **only perform read operations**. Writes should be performed
+in the independent trial directories. Here are two ways of accomplishing this:
+
+
+Option 1: Setting `chdir_to_log_dir=False` in `air.RunConfig`
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+The first option is to explicitly tell Tune to not change the working directory
+to the trial directory. In this case, the :meth:`session.get_log_dir() <ray.air.session.get_log_dir>`
+API should be used to get the path for writes.
+
+.. literalinclude:: doc_code/faq.py
+    :dedent:
+    :emphasize-lines: 9, 15
+    :language: python
+    :start-after: __no_chdir_start__
+    :end-before: __no_chdir_end__
+
+Option 2: Manual control using environment variables
+''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+The second option is to allow Tune to change the working directory and use the
+``RAY_ORIG_WORKING_DIR`` environment variable to convert relative paths to the
```

Comment:
Why don't we add this to the `session` object and deprecate the env vars altogether? It seems inconsistent to have some metadata in env vars and some in the session object.

Reply:
I was considering this but didn't know whether this 2nd option was too niche of a use case to introduce a high level `session` API for it.

Reply:
Should we deprecate the environment variable and only recommend option 1 of setting the chdir flag to False? I'm just worried about cluttering the `session` API with `get_orig_working_dir` that might not be as needed as something like `get_log_dir`. cc @krfricke

Reply:
Yes, I think we should do this. There shouldn't be multiple ways of getting session metadata. Also, this is much more discoverable than an env var.